config.go

```diff
@@ -735,6 +757,10 @@ func loadConfig() (*config, []string, error) {
 	// Limit the block priority and minimum block sizes to max block size.
 	cfg.BlockPrioritySize = minUint32(cfg.BlockPrioritySize, cfg.BlockMaxSize)
 	cfg.BlockMinSize = minUint32(cfg.BlockMinSize, cfg.BlockMaxSize)
+	cfg.BlockMinWeight = minUint32(cfg.BlockMinWeight, cfg.BlockMaxWeight)
+
+	// TODO(roasbeef): need to do multi-dimensional limiting based on if
```

Comment:
Added those two cases, i think with just then all cases are addressed as: if both are set then the user has a particular policy they're trying to enforce, if non are set, then the defaults will be used.